.jvmopts

```diff
@@ -1,4 +1,5 @@
 -Dfile.encoding=UTF8
+-Djava.security.egd=file:/dev/./urandom
```

Comment:
Looking a bit closer I'm not sure this will have the intended effect: it will add `-Djava.security.egd=file:/dev/./urandom` to the JVM running `sbt-launch.jar`, but [tests run in a forked JVM](https://github.com/akka/alpakka/blob/master/build.sbt#L103) and it looks like this parameter is not passed on to the forked JVM. Perhaps we should configure this parameter somewhere in sbt instead to make sure it is passed to the forked JVM? Also it might be good to mention this issue in the documentation for this module.